src/terrain/terrain_base.cpp

```diff
@@ -69,8 +69,8 @@ TerrainBase::TerrainBase(
     );
 
     // TODO make this faster 4
-    for (int i = 0; i < x; i++)
-        for (int j = 0; j < y; j++) {
+    for (size_t i = 0; i < x; i++)
+        for (size_t j = 0; j < y; j++) {
```

Comment:
Make this a for-each loop

Reply:
what how? I didn't know this was possible in c++. I though a foreach loop had to be over an iterable and not an int.

Reply:
Oh woops misread